_data/rules.yml

```diff
@@ -89,11 +89,11 @@ services:
             severity: warning
           - name: Container Memory usage
             description: Container Memory usage is above 80%
-            query: '(sum(container_memory_usage_bytes) BY (ip) / sum(container_spec_memory_limit_bytes) BY (ip) * 100) > 80'
+            query: '(sum(container_memory_usage_bytes) BY (ip) / sum(container_memory_max_usage_bytes) BY (ip) * 100) > 80'
             severity: warning
           - name: Container Volume usage
             description: Container Volume usage is above 80%
-            query: '(sum(container_fs_inodes_total) BY (ip) / sum(container_fs_inodes_total) BY (ip) * 100) > 80'
+            query: '(sum(container_fs_inodes_free) BY (ip) / sum(container_fs_inodes_total) BY (ip) * 100) > 80'
```

Comment:
It should be `(1 - (sum(container_fs_inodes_free) BY (ip) / sum(container_fs_inodes_total) BY (ip)) * 100) > 80`

Reply:
Nice :D